### DISCUSSION TITLE
Announcing Blazorise new features

### DISCUSSION THREAD

**Discussion by Stirda** [abpframework/abp#9738] [Ideas] [answered]
It is a pity that Blazorise new features are so hard to find. v0.9.4 give cool additions but I did not find any changelog :

- @stsrki Blazorise GitHub: no release "0.9.4".
- @stsrki Blazorise GitHub: no tag "0.9.4".
- @stsrki Blazorise documentation: no release "0.9.4".
- @hikalkan ABP 4.4 announcement in ABP blog: no mention to Blazorise new features (neither RC version nor final version).

**Comment by stsrki** [comment, marked as answer]
Hi @Stirda, all Blazorise release notes can be seen at news sections: https://blazorise.com/news/

**Reply by Stirda** [reply]
Oh, great! Thank you.
